project-base/app/src/DataFixtures/Performance/ProductDataFixture.php

```diff
@@ -115,9 +113,7 @@ public function load(OutputInterface $output)
                 $this->productsByCatnum[$product->getCatnum()] = $product;
             }
 
-            if ($this->countImported % self::BATCH_SIZE === 0) {
-                $this->cleanAndLoadReferences();
-            }
+            $this->cleanAndLoadReferences();
```

Comment:
is not that an overkill to do so after each product? I would expect it would be better just to decrease the `BATCH_SIZE` value, or? Have you tried changing it to `10`, `100` maybe?

Reply:
This leads to much faster generation than cleaning after any batch. I have tried several scenarios and this wast the fastest one.

Reply:
ok, thanks for the info